addons/mail/static/src/core/common/attachment_list.js

```diff
@@ -102,6 +102,10 @@ export class AttachmentList extends Component {
         });
     }
 
+    onAttachmentClicked(attachment) {
+        this.fileViewer.open(attachment, this.props.attachments);
+    }
+
```

Comment:
Please name it `onClickX` rather than `onXClicked`. The former is much easier to read and grasp that this is a handler function of a `t-on-click`